src/main/java/org/fundacionjala/at15/pokemon/Trainer.java

```diff
@@ -0,0 +1,39 @@
+package org.fundacionjala.at15.pokemon;
+
+public class Trainer {
+    private final int limit = 6;
```

Comment:
PokemonTeam class will controll this limit

Reply:
Yes, it's just for now, until the TeamPokemon will be merged